crda/remotematcher.go

```diff
@@ -0,0 +1,258 @@
+package crda
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"golang.org/x/sync/errgroup"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/rs/zerolog"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/libvuln/driver"
+)
+
+var (
+	_ driver.Matcher       = (*Matcher)(nil)
+	_ driver.RemoteMatcher = (*Matcher)(nil)
+)
+
+const (
+	// Bounded concurrency limit.
+	defaultRequestConcurrency = 10
+	defaultURL                = "https://f8a-analytics-preview-2445582058137.production.gw.apicast.io/?user_key=3e42fa66f65124e6b1266a23431e3d08"
```

Comment:
Is this okay for the default for everyone that'll ever use this? It seems odd to have "preview" in the subdomain and a "user_key" URL parameter.

Reply:
@hdonnay I had used preview/staging deployment to avoid messing up with our reports. 
"user_key" is not a secrete key, it is a publicity known 3scale gateway key.